evadb/executor/load_executor.py

```diff
@@ -30,6 +31,9 @@ def exec(self, *args, **kwargs):
         """
 
         # invoke the appropriate executor
+        if self.node.file_options["file_format"] is None:
```

Comment:
Please loop over all the possible enum values so that we don’t need to update this if we add more types later. Thanks!

Reply:
@gaurav274 , in this if condition we are checking if we are receiving the file format none. It would be the case only if the file format is not their in lark parser.
And after this there is a condition already available where we are checking all the available file format and passing it to the load executor